assets/js/blocks/cart-checkout/cart/index.js

```diff
@@ -51,4 +51,6 @@ const settings = {
 	},
 };
 
-registerBlockType( 'woocommerce/cart', settings );
+if ( WOOCOMMERCE_BLOCKS_PHASE > 1 ) {
```

Comment:
Do you think it would be useful to add a util function for this that supports named properties? e.g. `is_build( 'experimental' )`

Reply:
yeah now that we don’t have dead code elimination this could be useful, and it could help enforce tighter guidelines like throwing an error if the flag you provided doesn’t exist.

Reply:
I think if we're going to add convenience utility functions, it'd be nicer to do something like `isExperimentalBuild()` or `isStableBuild()`.

Reply:
where should those methods live 🤔

Reply:
> where should those methods live

Good question! Maybe block settings (since that's where `WOOCOMMERCE_BLOCKS_PHASE` comes from)?